app/views/shared/partials/_restrictions_form.html.erb

```diff
@@ -0,0 +1,52 @@
+<% list_of_restrictions = (@construction.payload || {})["restrictions"].presence || [OpenStruct.new] %>
+
+<div class="row">
+  <div class="col">
+    <h3 class="d-sm-flex align-items-center justify-content-between my-4">
+      Eingrenzung
+    </h3>
+  </div>
+</div>
+
+<div id="nested-restrictions">
+  <% list_of_restrictions.each_with_index do |restriction, index| %>
+    <%= fields_for "#{record_type}[payload][restrictions][#{index}]", restriction do |fp|  %>
```

Comment:
`fp` scheint kopiert vom `price_informations_form` und könnte hier ruhig `fr` sein für `form restriction`

Reply:
done in 67379560fa7b13df85df2ea1e62c2854ca50048d